<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialStatusBar</class>
 <widget class="QWidget" name="MaterialStatusBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>543</width>
        <height>156</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="StatusBarStyleSheet">
         <property name="styleSheet">
          <string notr="true">QFrame
{
	background-color: rgb(64, 64, 64);
	border: 1px solid black;
}

QPushButton
{
	background-color: rgb(136, 136, 136);
	color: white;
	border: 1px solid black;
    padding: 4px;
}

QPushButton:pressed 
{
    background-color: rgb(64, 64, 64);
}

QLabel
{
	color: white;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="StatusBarHorLayout">
            <item>
             <widget class="QLabel" name="labelMatStatus">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>12</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border: 0px solid black;</string>
              </property>
              <property name="text">
               <string>Status: 0 Active / 0 Completed / 0% Complete</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="addMatPushButton">
              <property name="text">
               <string>Add Material</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="searchMaterialLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="matTrackerToolBarLayout"/>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
